onos-config/templates/secret.yaml

```diff
@@ -2,6 +2,23 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
+{{- if .Values.certManager.enabled }}
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: {{.Values.certManager.name}}
+  namespace: {{.Values.certManager.namespace}}
+spec:
+  dnsNames:
+    - certs.oie.intel.com
```

Comment:
Please remove any references to intel stuff.

Reply:
done